src/components/ArticleCard.jsx

```diff
@@ -2,10 +2,11 @@ const Article = ({ article }) => {
   return (
     <>
       <h3>{article.title}</h3>
-      <p>{article.author}</p>
 
       <span>{article.topic}</span>
       <span>{article.created_at}</span>
+      {!article.body ? "" : <p className="text-body">{article.body}</p>}
```

Comment:
You could use double ampersand here if you don't have a second condition instead of putting in an empty string (ie. `{article.body && <p>body here</p> }` )

Reply:
Thanks, really helpful :)